idaes/surrogate/tests/test_keras_surrogate.py

```diff
@@ -15,19 +15,20 @@
 """
 import pytest
 
+pytest.importorskip("tensorflow.keras", reason="tensorflow.keras not available")
+pytest.importorskip("omlt", reason="omlt not available")
+
```

Comment:
I am surprised that this works with Pyomo's attempt_import stuff. Is there some magic going on under the hood?